static/app/views/performance/http/httpSamplesPanel.tsx

```diff
@@ -254,23 +278,35 @@ export function HTTPSamplesPanel() {
             </SegmentedControl>
           </ModuleLayout.Full>
 
-          <ModuleLayout.Full>
-            {query.panel === 'duration' && (
-              <DurationChart
-                series={durationData[`avg(span.self_time)`]}
-                isLoading={isDurationDataFetching}
-                error={durationError}
-              />
-            )}
+          {query.panel === 'duration' && (
+            <Fragment>
+              <ModuleLayout.Full>
+                <DurationChart
+                  series={durationData[`avg(span.self_time)`]}
+                  isLoading={isDurationDataFetching}
+                  error={durationError}
+                />
+              </ModuleLayout.Full>
 
-            {query.panel === 'status' && (
+              <ModuleLayout.Full>
+                <SpanSamplesTable
+                  data={samplesData}
+                  isLoading={isSamplesDataFetching}
```

Comment:
m: I think this should also depend on `isDurationDataFetching` considering `samplesData` is dependent on `durationData`.

I'm noticing that `SpanSamplesTable` is going from empty state -> loading instead of starting at loading, which is why I'm suggesting this here.

Reply:
Fixed, thanks!